src/riak_pb_ts_codec.erl

```diff
@@ -124,6 +124,8 @@ encode_cell(V) when is_boolean(V) ->
 encode_cell({time, V}) ->
     #tscell{timestamp_value = V};
 encode_cell(undefined) ->
+    #tscell{};
+encode_cell([]) -> %% NULL Cell
```

Comment:
@alexmoore what is the case where `[]` is the value? Is it on data insertion or retrieval?